code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -81,63 +81,63 @@
 
 //Pills
 /obj/item/weapon/reagent_containers/pill/antitox
-	name = "Anti-toxins pill"
+	name = "Anti-toxins pill (25u)"

```

Comment:
This is a really iffy way of doing it imo, a better way would be to add onto the name in Initialize() with the total num of reagents in it.

Reply:
Which shouldn't be a manual entry! That'd just lead to people forgetting to update it in the off chance it's changed.

I think there's a proc to get the total reagents in any specific object.

Reply:
As the newbie i am, where could i figure out where such proc exist?

Reply:
You'd want to check
/obj/item/weapon/reagent_containers/pill/Initialize()

You'd probably run into the issue of it Initializing before reagents are added, however. Easiest/Shittiest way would be throwing a timer in the Initialize, but there's probably a better way to do it.

I'd suggest asking around the Code dungeon and finding someone who knows what they're doing.